tensorflow/lite/micro/tools/make/targets/ceva_bx1_makefile.inc

```diff
@@ -1,9 +1,8 @@
 
-ifeq ($(TARGET), ceva)
-ifeq ($(TARGET_ARCH), bx1)
- #TARGET_ARCH := 
+ifeq ($(TARGET), ceva_bx1)
+TARGET_ARCH := bx
 CXXFLAGS := -std=c++11 -DTF_LITE_STATIC_MEMORY
-CCFLAGS  := -std=c11   -DTF_LITE_STATIC_MEMORY
+CCFLAGS  := -std=c11   -DTF_LITE_STATIC_MEMORY -I./tensorflow/lite/micro/kernels/ceva_bx1
```

Comment:
preference would be to use full include paths, unless there is a strong reason why that would not work.